sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 12.0.2 (Unreleased)
+
+- Improvements to sample code.
+- Update tracing to set standard span attribute [PR #6565](https://github.com/Azure/azure-sdk-for-js/pull/6565)
+- Share a single httpClient among clients. [PR #6657](https://github.com/Azure/azure-sdk-for-js/pull/6657)
```

Comment:
Does this provide any benefits to the user or was it an internal refactoring to re-use the clients?
If there is no net benefit or affect to the end user, then we can skip it from the changelog.

Reply:
Not really impacting consumer of this lib. Removed.